projects/driver_controls/inc/stm32f0xx/ads1015_def.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
```

Comment:
Please add comments explaining where to find the information you have here and a module-level comment explaining that this is an internal file.